lib/number.js

```diff
@@ -53,12 +53,29 @@ internals.Number = class extends Any {
 
     multiple(base) {
 
-        Hoek.assert(Hoek.isInteger(base), 'multiple must be an integer');
```

Comment:
If you don't remove this then it throws error instead of gracefully validating (thats how it happens even now). It starts throwing exception for non-integer number, so what is the point in allowing float numbers?